common/src/main/java/com/wynntils/models/lootrun/LootrunModel.java

```diff
@@ -1233,4 +1258,76 @@ private void parseFailedMessages(StyledText styledText) {
     private LootrunDetails getCurrentLootrunDetails() {
         return lootrunDetailsStorage.get().getOrDefault(Models.Character.getId(), new LootrunDetails());
     }
+
+    @SubscribeEvent
+    public void onConfirmationClick(ContainerClickEvent event) {
+        if (!(Models.Container.getCurrentContainer() instanceof LootrunRewardContainer container)) return;
+
+        Component itemName = event.getItemStack().getCustomName();
```

Comment:
```suggestion
        Component itemName = event.getItemStack().getHoverName();
```

I don't think there's any real difference but we tend to use this